app/controllers/api/account.php

```diff
@@ -2300,6 +2302,7 @@
             ->setFrom($from)
             ->setRecipient($user->getAttribute('email'))
             ->setName($user->getAttribute('name'))
+            ->setSmtp($project->getAttribute('smtp', []))
```

Comment:
Same suggestions about the complex structure.

Reply:
updated